src/dbally/views/sqlalchemy_base.py

```diff
@@ -27,6 +28,15 @@ def get_select(self) -> sqlalchemy.Select:
         which will be used to build the query.
         """
 
+    def get_subquery(self) -> sqlalchemy.Subquery:
+        """
+        Creates the initial sqlalchemy.Subquery object, which will be used to build the query.
+
+        Returns:
+            The sqlalchemy.Subquery object based on private _select attribute.
```

Comment:
I don't think we should mention private elements (i.e., internal implementation details) in public documentation. On the other hand, I'm not sure the end-users need `get_subquery` at all, so maybe it should be marked as private as well (or removed and folded into the `apply_aggregation` method itself).